pkg/services/ngalert/api/tooling/definitions/provisioning_mute_timings.go

```diff
@@ -107,24 +103,34 @@ type MuteTimeIntervalExport struct {
 	config.MuteTimeInterval `json:",inline" yaml:",inline"`
 }
 
-// MuteTimeIntervalExportHcl is a representation of the MuteTimeInterval in HCL
-type MuteTimeIntervalExportHcl struct {
-	Name          string                  `json:"name" hcl:"name"`
-	TimeIntervals []TimeIntervalExportHcl `json:"time_intervals" hcl:"intervals,block"`
+// MuteTimeIntervalAPIModel is a serialized representation of the MuteTimeInterval struct, as returned by the API
+// swagger:model
+type MuteTimeIntervalAPIModel struct {
+	MuteTimeIntervalModel
+	Provenance Provenance `json:"provenance,omitempty"` // TODO: This is never populated on GETs
+}
+
+// swagger:model
+type MuteTimingsAPIModel []MuteTimeIntervalAPIModel
+
+// MuteTimeIntervalModel is a serialized representation of the MuteTimeInterval struct
+type MuteTimeIntervalModel struct {
```

Comment:
Let's rename it to `TimeInterval`. Mute time interval is deprecated model in Alertmanager and is replaced with TimeInterval. We are going to migrate to it

Reply:
The issue with naming it `TimeInterval` is that there's already one in the spec: https://github.com/grafana/grafana/blob/main/pkg/services/ngalert/api/tooling/spec.json#L7521. It comes from `grafana/prometheus-alertmanager.timeinterval` and it seems to have priority.

Reply:
Let's call it TimeIntervals then? Or NamedTimeIntervals.